packages/manager/cypress/e2e/core/objectStorageGen2/bucket-object-gen2.spec.ts

```diff
@@ -364,4 +366,110 @@ describe('Object Storage Gen2 bucket object tests', () => {
 
     checkBucketObjectDetailsDrawer(bucketFilename, endpointTypeE3);
   });
+
+  it('UI should display buckets for one region, warning message for one region', () => {
```

Comment:
```suggestion
  it('displays successfully fetched buckets , warning message for failed fetch', () => {
```
Just a nitpick! And maybe something similar for the other test's name